more/free-programming-interactive-tutorials-en.md

```diff
@@ -379,6 +380,11 @@
 * [Rustlings](https://github.com/rust-lang/rustlings)
 
 
+### Salesforce
+
+* [Salesforce Solutioning] (https://www.pluralsight.com/courses/play-by-play-diagramming-salesforce-solutions by Matthew Morris and Don Robins)
```

Comment:
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.